environment.yml

```diff
@@ -0,0 +1,21 @@
+name: UPenn_Holdings_Project
+channels:
+- anaconda-fusion
+- defaults
+dependencies:
+- beautifulsoup4=4.6.0
```

Comment:
I've done this in b66a9931c. Does that look correct to you? The only mention I've found so far re: which channel a package is from is by doing, e.g., `conda search pandas`, which lists `defaults` as the channel. So that's what I've used here. Does that jibe with your understanding?